src/main/kotlin/lotto/domain/number/FixedNumberGenerator.kt

```diff
@@ -0,0 +1,8 @@
+package lotto.domain.number
+
+class FixedNumberGenerator : NumberGenerator {
+
+    override fun generate(count: Int): IntArray {
+        return IntArray(count) { it -> it + 1 }
+    }
```

Comment:
```suggestion
    override fun generate(count: Int) = IntArray(count) { it + 1 }
```

could be also written like this